haystack/preview/components/fetchers/link_content.py

```diff
@@ -8,9 +8,9 @@
 from requests.exceptions import HTTPError
 from tenacity import RetryCallState, retry, retry_if_exception_type, stop_after_attempt, wait_exponential
 
-from haystack import __version__
 from haystack.preview import component, default_from_dict, default_to_dict
 from haystack.preview.dataclasses import ByteStream
+from haystack.preview.version import __version__
```

Comment:
I'm wondering why `LinkContentFetcher` needs Haystack's version

Reply:
It's used in the user agent at [line 18](https://github.com/deepset-ai/haystack/pull/6052/files#diff-4a5e0e4c815437b7362f61d07e2ed8fcc1cba9275227c5e0f6466c44e1f4c6b3R18).